### PR TITLE
refactor: termination handler

### PR DESCRIPTION
**Pull request by Duologic**
* Do not wrap everything up in new()
* Use k.libsonnet import
* Build upon k8s-alpha

Note: needs https://github.com/jsonnet-libs/k8s/pull/6

### CODE REVIEW

**Comment by Duologic** [comment]
The individual commits should make it easier to review.

**Comment by Duologic** [comment]
rebased.

File: k8s-node-termination-handler/handler.libsonnet [hunk 1]
```diff
@@ -1,105 +1,9 @@
...
-      clusterRoleBinding.mixin.roleRef.withKind('ClusterRole') +
-      clusterRoleBinding.mixin.roleRef.withName('node-termination-handler-role') +
-      clusterRoleBinding.withSubjectsMixin({
-        kind: 'ServiceAccount',
-        name: 'node-termination-handler',
-        namespace: _config.namespace,
-      }),
-  },
+    } + config;
+    (import 'gke.libsonnet'),
```
Comment (malcolmholmes) [inline k8s-node-termination-handler/handler.libsonnet:8]: is this line for back-compat?

Comment (Duologic) [reply k8s-node-termination-handler/handler.libsonnet:8]: yes: https://github.com/grafana/jsonnet-libs/pull/286/commits/11997d1e0623bf09cd83d0679ee24e901f1285a6

Comment (Duologic) [reply k8s-node-termination-handler/handler.libsonnet]: mistake made, new commit added

**Review by malcolmholmes — APPROVED** [review]

**Pull request merged**